src/main/java/org/commonjava/indy/service/tracking/model/AccessChannel.java

```diff
@@ -29,4 +29,9 @@
     /** Used to signify content coming from normal repositories and groups. */
     NATIVE,
 
+    /** Used when the store is accessed via regular Maven repo.
+     *  NOTE: This has been changed to {@link #NATIVE} in our tracking code. It is included for historical purposes. */
+    @Deprecated
+    MAVEN_REPO;
```

Comment:
Why is this needed? Is this causing some errors?

Reply:
yes, old records have an access channel set to this and when constructing those records it throws an error so we need this.

Reply:
Ok. Let's leave it here.